CHANGELOG.md

```diff
@@ -17,152 +19,222 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## [6.12.0] - 2023-07-26
+
+### Added
+
+* Added option in `.to_pandas()` method to include expanding metadata columns into a dataframe as well as an option to prefix those columns. Default is to prepend ".metadata" and `expand_metadata` defaults to `False` in order to maintain backwards compatibility.
+* Updated installation process for contributing.
```

Comment:
The dot should come after metadata I think. I also took the liberty of rewriting slightly 😉 
```suggestion
- Added option `expand_metadata` to `.to_pandas()` method for list resource types which converts the metadata (if any) into separate columns in the returned dataframe. Also added `metadata_prefix` to control the naming of these columns (default is "metadata.").
```